src/main/java/org/opensearch/performanceanalyzer/collectors/AdmissionControlMetricsCollector.java

```diff
@@ -57,8 +60,11 @@ public void collectMetrics(long startTime) {
 
         long startTimeMillis = System.currentTimeMillis();
         try {
-            Class admissionController = Class.forName(ADMISSION_CONTROLLER);
-            Class jettyAdmissionControlService = Class.forName(ADMISSION_CONTROL_SERVICE);
+            Class admissionController =
+                    Class.forName(ADMISSION_CONTROLLER, false, this.admissionControlClassLoaer);
+            Class jettyAdmissionControlService =
+                    Class.forName(
+                            ADMISSION_CONTROL_SERVICE, false, this.admissionControlClassLoaer);
```

Comment:
This will load the class `ADMISSION_CONTROLLER`, `ADMISSION_CONTROL_SERVICE`  each time `collectMetrics` is executed (5s granularity). 

Instead, can this operation be done within the `AdmissionControlMetricsCollector` constructor?

Reply:
Makes sense.